app/utils/geostore.py

```diff
@@ -34,7 +40,23 @@ async def _get_gfw_geostore(geostore_id: UUID) -> GeostoreCommon:
     return geostore_common
 
 
-async def get_geostore(geostore_id: UUID, geostore_origin: str) -> GeostoreCommon:
+def check_all_geostores():
+    return FEATURE_CHECK_ALL_GEOSTORES == "TRUE"
+
+
+async def get_geostore(
+    geostore_id: UUID, geostore_origin: GeostoreOrigin
+) -> GeostoreCommon:
+    if check_all_geostores():
+        return await get_geostore_from_any_origin(geostore_id, geostore_origin)
+    else:
+        return await get_geostore_legacy(geostore_id, geostore_origin)
+
+
+async def get_geostore_legacy(
```

Comment:
Instead of having a function named legacy and function that gets from any origin, could we just have one function that accepts a list of origins and searches for them in that order? For the legacy function, just only send one origin.